packages/loader/container-loader/src/loader.ts

```diff
@@ -53,6 +53,13 @@ function canUseCache(request: IRequest): boolean {
 	return request.headers[LoaderHeader.cache] !== false;
 }
 
+function ensureFluidResolvedUrl(
```

Comment:
This is just an undefined check now, suggest inline.

Reply:
for some reason i thought it had a few more usage, which made this cheaper in terms of code characters over the wire, but looks like i found a way around the other checks.

Reply:
IMO the few bits aren't worth the extra indirection (if it even saves bits - inlining will save its own few bits as well).  These little mental taxes hinder folks new to the code from getting productive - in isolation they seem small but they really add up.

If you do keep this however, please rename to reflect its actual behavior (assertNotUndefined or similar).  Again strongly suggest using strict boolean comparisons esp. for arguments that could easily be construed as string-like (and therefore truthy-ambiguous) for clarity.

On a related note, this probably points to another failing of the interface - are there any places in our code that are actually OK with a url resolver returning undefined?  MultiUrlResolver used it as part of its fallback logic but now that's gone.  Another thing worth revisiting in the url resolution space.

Reply:
i renamed it. i don't think i agree with everything you've said, but this we can agree to disagree with the rename.